src/Core/src/Platform/Android/MauiAppCompatActivity.cs

```diff
@@ -40,21 +39,99 @@ protected override void OnCreate(Bundle? savedInstanceState)
 
 			CoordinatorLayout parent = new CoordinatorLayout(this);
 
-			SetContentView(parent, new ViewGroup.LayoutParams(CoordinatorLayout.LayoutParams.MatchParent, CoordinatorLayout.LayoutParams.MatchParent));
+			SetContentView(parent, new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MatchParent, ViewGroup.LayoutParams.MatchParent));
 
 			//AddToolbar(parent);
 
-			parent.AddView(content.ToNative(window.MauiContext), new CoordinatorLayout.LayoutParams(CoordinatorLayout.LayoutParams.MatchParent, CoordinatorLayout.LayoutParams.MatchParent));
+			parent.AddView(content.ToNative(window.MauiContext), new CoordinatorLayout.LayoutParams(ViewGroup.LayoutParams.MatchParent, ViewGroup.LayoutParams.MatchParent));
+
+			foreach (var androidLifecycleHandler in GetAndroidLifecycleHandler())
+				androidLifecycleHandler.OnCreate(this, savedInstanceState);
+		}
+
+		protected override void OnStart()
```

Comment:
Do we want to use the overrides here or make use of the AndroidX lifecycle attributes feature? Is there a benefit to either one?